### PR TITLE
Remove old tests

### PR DESCRIPTION
**Pull request by johnktims**

### CODE REVIEW

File: querydsl-sql/src/test/java/com/querydsl/sql/PerformanceTest.java [hunk 1]
```diff
@@ -1,67 +0,0 @@
...
-
-        iterations =  1000000;
-    }
-
-    @Test
-    public void nonNormalized() {
-        long start = System.currentTimeMillis();
-        for (int i = 0; i < iterations; i++) {
-            SQLSerializer serializer = new SQLSerializer(configuration);
-            serializer.setNormalize(false);
```
Comment (johnktims) [inline querydsl-sql/src/test/java/com/querydsl/sql/PerformanceTest.java]: This setter doesn't do anything anymore.

Comment (Shredder121) [inline querydsl-sql/src/test/java/com/querydsl/sql/PerformanceTest.java]: Oh yeah, you're right.

File: querydsl-sql/src/test/java/com/querydsl/sql/PerformanceTest.java [hunk 2]
```diff
@@ -1,67 +0,0 @@
...
-            serializer.serialize(md, false);
-            serializer.getConstants();
-            serializer.getConstantPaths();
-            assertNotNull(serializer.toString());
-        }
-        System.err.println("non-normalized " + (System.currentTimeMillis() - start));
-    }
-
-    @Test
-    public void default_() {
```
Comment (johnktims) [inline querydsl-sql/src/test/java/com/querydsl/sql/PerformanceTest.java]: This is a duplicate of `com.querydsl.sql.QueryPerformanceTest#serialization`.

File: querydsl-sql/src/test/java/com/querydsl/sql/QueryPerformanceTest.java [hunk 3]
```diff
@@ -13,12 +13,9 @@
 import com.querydsl.core.DefaultQueryMetadata;
 import com.querydsl.core.JoinType;
 import com.querydsl.core.QueryMetadata;
-import com.querydsl.core.testutil.Benchmark;
-import com.querydsl.core.testutil.Performance;
-import com.querydsl.core.testutil.Runner;
-import com.querydsl.core.testutil.Derby;
+import com.querydsl.core.testutil.*;
```
Comment (Shredder121) [inline querydsl-sql/src/test/java/com/querydsl/sql/QueryPerformanceTest.java]: How many imports are used from this package?
Just wondering.

Comment (johnktims) [inline querydsl-sql/src/test/java/com/querydsl/sql/QueryPerformanceTest.java]: Just 4. Isn't 3 our cutoff?

Comment (Shredder121) [inline querydsl-sql/src/test/java/com/querydsl/sql/QueryPerformanceTest.java]: It should be 5.
3 is the cutoff for static imports.

**Comment by Shredder121** [comment]
Nice, that's clean again.

**Comment by Shredder121** [comment]
This looks now good to me.

**Pull request merged**